### ISSUE TITLE
Guide on how to use the api

### ISSUE THREAD

**Issue by dsyenket** [ammezie/book-reviews-api#1]
Hi, I've followed all the steps and created the API but what is the parameters that all the endpoints accept? For example for register, I've passed in the header "Accept: application/json", what else do I need?

**Comment by nullwriter** [comment]
Bump...

**Comment by King-maker** [comment]
yeah I do agree the guy needs to explain more, like how to test the api with postman as it seems like a really valid tutorial

**Comment by ozombo** [comment]
@King-maker please refer to the updated readme to understand better. 
@dsyenket did you read and understand the tutorial for this repo?

**Comment by hendisantika** [comment]
For this requirement,
I have just create POSTMAN API to make it easier to test.

You can check in on this link --> https://www.getpostman.com/collections/a3ce70e703d65914effd

Sorry, I created the request is Indonesia. You can change it to English or any other language @dsyenket & @tngeene.

**Comment by hendisantika** [comment]
I also send the POSTMAN collection file as PR --> https://github.com/ammezie/book-reviews-api/pull/4
